deployment/terraform/create.go

```diff
@@ -181,8 +220,34 @@ func (t *Terraform) setupAppServers(output *terraformOutput, extAgent *ssh.ExtAg
 	}
 }
 
+func (t *Terraform) setupLoadtestAgents(extAgent *ssh.ExtAgent, output *terraformOutput) error {
+	// Creating the sysadmin user.
+	// should wait the server to start
+
+	for i := 0; i < len(output.Agents.Value); i++ {
+		val := output.Agents.Value[i]
+
+		if err := t.runAgent(extAgent, val.PublicIP); err != nil {
+			mlog.Error("error while setting up an agent", mlog.String("agent name", val.Tags.Name), mlog.Err(err))
+			continue
+		}
+	}
+
+	if err := t.updateCoordinatorConfig(extAgent, output); err != nil {
+		mlog.Error(err.Error())
```

Comment:
Let's return an error instead of logging here ? You are not following your own suggestions :)

Reply:
Yeah, it's like a "the shoemaker's son always goes barefoot" 😄